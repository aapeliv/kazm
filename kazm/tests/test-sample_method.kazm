class Example {
  int a; double b;

  void print_b() { double_println(me.b); }
  int add_with_a(int number) {
    return me.a + number;
  }
};

double get_b(Example e) { return e.b; }

int main() {
  Example ex; ex.a = 3; ex.b = 7.;
  int_println(ex.add_with_a(10)); // 13
  double_println(get_b(ex)); // 7.000000
}
