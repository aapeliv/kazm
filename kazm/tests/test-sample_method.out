13
7.000000
